labonneboite/importer/jobs/extract_etablissements.py

```diff
@@ -265,7 +265,7 @@ def get_offices_from_file(self):
                     siret, raisonsociale, enseigne, codenaf, numerorue, \
                         libellerue, codecommune, codepostal, email, tel, \
                         trancheeffectif_etablissement, _, _, _, \
-                        website1, website2, better_email, better_tel, \
+                        website1, website2, _, better_tel, \
```

Comment:
en passant je suggère de renommer `better_tel` en `tel_from_offers` (ce tel vient bien des offres non?)

typos dans le titre du commit => `ignore email from offers in office extract`